charts/plugin-br-pix-switch/templates/plugin-br-pix-switch/deployment.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "plugin-br-pix-switch.fullname" . }}
+  labels:
+    {{- include "plugin-br-pix-switch.labels" (dict "context" . "name" .Values.pixSwitch.name ) | nindent 4 }}
+spec:
+  revisionHistoryLimit: {{ .Values.pixSwitch.revisionHistoryLimit | default 10 }}
+  strategy:
+    {{- toYaml .Values.pixSwitch.deploymentStrategy | nindent 4 }}
+  replicas: {{ .Values.pixSwitch.replicaCount }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)